code401challenges/src/main/java/code401challenges/hashtable/RepeatedWord.java

```diff
@@ -1,33 +1,19 @@
 package code401challenges.hashtable;
 
-import com.google.common.base.Strings;
-
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.Set;
-
 public class RepeatedWord {
-
-    public static void main(String[] args) {
-        System.out.println();
-    }
-
-
-    public  HashTable repeatedWord(String str) {
-        HashTable ht = new HashTable(1024);
+    public static String repeatedWord(String str) {
+        HashTable<String, String> ht = new HashTable<>(1024);
         String[] splitWords = str.split(" ");
-        Set<Strings>  words = new HashSet(Arrays.asList(splitWords));
-        int count = 0;
-        for(Strings word :  words) {
-            for(String temp : splitWords) {
-                if(word.equals(temp)) {
-                    ++count;
-                }
+        for(String word :  splitWords) {
+            word = word.replaceAll("[^a-zA-Z]", "").toLowerCase();
```

Comment:
`.replaceAll()` has a runtime of `n`. If you use `.replaceAll()` on the entire string before you split it this will increase the efficiency of the algorithm